autoload/lsp/lsp.vim

```diff
@@ -497,7 +497,12 @@ export def AddServer(serverList: list<dict<any>>)
       return
     endif
 
-    var lspserver: dict<any> = lserver.NewLspServer(server.path, args)
+    if !server->has_key('sync')
+      server['sync'] = v:false
+    endif
+
+    var lspserver: dict<any> = lserver.NewLspServer(server.path, args, 
+          \ server.sync ? server.sync : v:false)
```

Comment:
As you are initializing server.sync to a default value of v:false above, do you need to check whether it is true here?

Reply:
correct, thats a mistake.. should have removed that when I made it default.